doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -248,6 +248,15 @@ Bluetooth samples
   * Fixed an issue where the sample would assert with the :kconfig:option:`CONFIG_ASSERT` Kconfig option enabled.
     This was due to calling the :c:func:`bt_iso_chan_send` function from a timer ISR handler and sending SDUs to the controller with invalid timestamps.
 
+* Added support for the nRF54LM20 PDK in:
+
+   * :ref:`peripheral_lbs` sample
+   * :ref:`central_uart` sample
+   * :ref:`peripheral_uart` sample
+   * :ref:`peripheral_hids_mouse` sample
+   * :ref:`peripheral_hids_keyboard` sample
+   * :ref:`bluetooth_central_hids` sample:
```

Comment:
```suggestion
* Added support for the nRF54LM20 PDK in the following samples:

   * :ref:`peripheral_lbs`
   * :ref:`central_uart`
   * :ref:`peripheral_uart`
   * :ref:`peripheral_hids_mouse`
   * :ref:`peripheral_hids_keyboard`
   * :ref:`bluetooth_central_hids`
```
Also place this as the first item under the Bluetooth samples heading.

Reply:
Done